sage-ios/SAGE/SAGE/RequestHoursView.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  RequestHoursView.swift
+//  SAGE
+//
+//  Created by Andrew on 10/27/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+import BSKeyboardControls
+
+class RequestHoursView: UIView {
+    
+    var dateField =  FormFieldItem()
+    var startTimeField = FormFieldItem()
+    var endTimeField = FormFieldItem()
+    var commentField = FormTextItem()
+    private var keyboardControls = BSKeyboardControls()
+    private var scrollView = UIScrollView()
```

Comment:
wait why does this need to be a scrollview? isn't the view always going to be smaller than the screen